docs/quickstarts/excel-quickstart-vue.md

```diff
@@ -181,29 +181,29 @@ After you complete the wizard, it creates a `My Office Add-in` folder, which con
 
 ## Start the dev server
 
-1. From the terminal, run the following command to start the dev server.
+1. Start the dev server.
 
    ```command&nbsp;line
    npm run serve
    ```
 
-2. In a web browser, navigate to `https://localhost:3000` (notice the `https`). If the page on `https://localhost:3000` is blank and without any certificate errors, it means that it is working. The Vue App is mounted after Office is initialized, so it only shows things inside of an Excel environment.
+2. In a web browser, navigate to `https://localhost:3000` (notice the `https`). If the page on `https://localhost:3000` is blank and without any certificate errors, it means that it's working. The Vue App is mounted after Office is initialized, so it only shows things inside of an Excel environment.
 
 ## Try it out
 
-1. Follow the instructions for the platform you'll be using to run your add-in and sideload the add-in within Excel.
+1. Run your add-in and sideload the add-in within Excel. Follow the instructions for the platform you'll be using:
 
    - Windows: [Sideload Office Add-ins on Windows](../testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)
    - Web browser: [Sideload Office Add-ins in Office on the web](../testing/sideload-office-add-ins-for-testing.md#sideload-an-office-add-in-in-office-on-the-web)
    - iPad and Mac: [Sideload Office Add-ins on iPad and Mac](../testing/sideload-an-office-add-in-on-ipad-and-mac.md)
 
-2. In Excel, choose the **Home** tab, and then choose the **Show Taskpane** button in the ribbon to open the add-in task pane.
+2. Open the add-in task pane in Excel. On the **Home** tab, choose the **Show Taskpane** button.
 
    ![Screenshot of the Excel Home menu, with the Show Taskpane button highlighted.](../images/excel-quickstart-addin-2a.png)
 
 3. Select any range of cells in the worksheet.
 
-4. In the task pane, choose the **Set color** button to set the color of the selected range to green.
+4. Set the color of the selected range to green. In the task pane, choose the **Set color** button.
```

Comment:
Not related, but let's make it more clear which task pane the user needs to go to.
```suggestion
4. Set the color of the selected range to green. In your add-in's task pane, choose the **Set color** button.
```

Reply:
The illustration following this step makes it clear.